src/osmedile/intellij/stringmanip/transform/DelimitedListDialog.java

```diff
@@ -0,0 +1,96 @@
+package osmedile.intellij.stringmanip.transform;
+
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import javax.swing.JCheckBox;
+import javax.swing.JDialog;
+import javax.swing.JEditorPane;
+import javax.swing.JPanel;
+import javax.swing.JRadioButton;
+import javax.swing.JTextField;
+import javax.swing.event.DocumentEvent;
+
+import org.jetbrains.annotations.NotNull;
+
+import com.intellij.openapi.editor.Editor;
+import com.intellij.ui.DocumentAdapter;
+
+import shaded.org.apache.commons.text.StringEscapeUtils;
+
+/**
+ * @author Philipp Menke
+ */
+class DelimitedListDialog extends JDialog {
+    private static final long                serialVersionUID = -9173380044265093635L;
+    private final        DelimitedListAction action;
+    private final        Editor              editor;
+
+    JPanel     contentPane;
+    JTextField tfDestDelimiter;
+    private JTextField   tfSourceDelimiter;
+    private JRadioButton rbSourceSelection;
+    private JRadioButton rbSourceClipboard;
+    private JCheckBox    cbAutoQuote;
+    private JTextField   tfQuote;
+    private JEditorPane  epPreview;
```

Comment:
I have preview editor in TextAlignmentForm#myPreviewPanel, it is probably much nicer, check it out

Reply:
And you will not need to limit the length